doc/release-notes/release-7.sgml.in

```diff
@@ -57,6 +58,23 @@ The Squid-@SQUID_RELEASE@ change history can be <url url="https://github.com/squ
 
 </descrip>
 
+<sect1>Removed purge tool
+<p>The <em>purge</em> tool (also known as <em>squidpurge</em>, and <em>squid-purge</em>)
+   was limited to managing UFS/AUFS/DiskD caches.
+
+<p>The cache contents display and search it provided can be obtained with a script
+   searching the cache manager <em>objects</em> report.
+
+<p>This tool used the custom <em>PURGE</em> HTTP method to remove cache
+   objects. This can be performed directly on any Squid configured to allow
+   the method. Like so:
+<verb>
+    acl PURGE method PURGE
+    http_access allow manager PURGE
+</verb>
```

Comment:
```suggestion
</verb>
    any HTTP client (such as curl) can then be used to evict objects from the cache, for isntance:
<verb>
$ curl -XPURGE --proxy http://squid.address:3128 --proxy-user admin:password http://url.to.evict/
</verb>
```

Reply:
I would drop the authentication part to simplify the example, especially since there are so many ways Squids do access controls:

```suggestion
</verb>
    any HTTP client (such as curl) can then be used to evict objects from the cache, for example:
<verb>
$ curl -XPURGE --proxy http://127.0.0.1:3128 http://url.to/evict/
</verb>
```